docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,194 @@
+# Host Settings
+This page contains settings for Linux servers, it agnostic of the Linux
+distribution. The commands use the `ip` utility from the `iproute2` package.
+These settings do not persist across reboots. Consult the documentation from
+the Linux distribution for guides on how to persist the settings in for example
+[Netplan](https://documentation.ubuntu.com/server/explanation/networking/configuring-networks/index.html) or 
+[Network Manager](https://networkmanager.dev/docs/admins/). For additional
+details consult the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation.
+
+## MCLAG / ESLAG
+
+The multi-chassis LAG architecture is a way to provide device redundancy
+in a network architecture. At the physical layer an MCLAG topology is a single
+server connected to two different switches and those switches are directly connected
+to each other in addition to being connected to the rest of the fabric.
+
+ESLAG is a similar topology as MCLAG with the beneficial difference that the
+switches do not need a to be connected to each other. There can be up to 4
+switches in an ESLAG group, whereas MCLAG is always 2 switches.
+
+The standard that governs this technology is commonly referred to as 802.3ad.
+
+### Server Settings
+
+The supported server side setting for bonding is `802.3ad`, with hashing policy
+`layer2` or `layer2+3`. To configure this in Linux:
```

Comment:
Do we want to recommend layer2?  Performance will suck for any routed TCP and similar traffic.  If we insist on mentioning it, we should say, something along the lines of `unless you have a specific reason, you should choose layer 2+3`.